packages/wpcom-template-parts/src/utils.ts

```diff
@@ -1,24 +1,30 @@
 import { translate as translateMethod } from 'i18n-calypso';
 
 export const getAutomatticBrandingNoun = ( translate: typeof translateMethod ) => {
+	const AutomatticBrand = 'Automattic';
+	const options = {
+		description:
+			'Branding to be shown on the Footer of the page, Automattic\'s variable will always contains the word "Automattic"',
+		args: { Automattic: AutomatticBrand },
+	};
 	const automatticRoger = [
-		translate( 'An Automattic brainchild' ),
-		translate( 'An Automattic contraption' ),
-		translate( 'An Automattic creation' ),
-		translate( 'An Automattic experiment' ),
-		translate( 'An Automattic invention' ),
-		translate( 'An Automattic joint' ),
-		translate( 'An Automattic medley' ),
-		translate( 'An Automattic opus' ),
-		translate( 'An Automattic production' ),
-		translate( 'An Automattic ruckus' ),
-		translate( 'An Automattic thingamajig' ),
+		translate( 'An %(Automattic)s brainchild', options ),
+		translate( 'An %(Automattic)s contraption', options ),
+		translate( 'An %(Automattic)s creation', options ),
+		translate( 'An %(Automattic)s experiment', options ),
+		translate( 'An %(Automattic)s invention', options ),
+		translate( 'An %(Automattic)s joint', options ),
+		translate( 'An %(Automattic)s medley', options ),
+		translate( 'An %(Automattic)s opus', options ),
+		translate( 'An %(Automattic)s production', options ),
+		translate( 'An %(Automattic)s ruckus', options ),
+		translate( 'An %(Automattic)s thingamajig', options ),
```

Comment:
This is a good idea, to turn "Automattic" into a variable. Having this, you could pass the SVG directly into the `translate` function:
```js
function AutomatticBrand() {
  return <>
    <span hidden>Automattic</span>
    <svg><path d="draw the letters"/></svg>
  </>;
}

function getAutomatticBranding( translate, brand ) {
  return translate( 'An %(Automattic) thing', { args: { Automattic: brand } } );
}

getAutomatticBranding( translate, <AutomatticBrand /> );
```
With interpolated translation string, you no longer need to split manually into prefix and suffix.

Reply:
I really liked this solution and tried to implement it, but apparently, when interpolating the translation with a markup it always generates a different signature.

<img width="1246" alt="CleanShot 2023-06-12 at 12 59 15@2x" src="https://github.com/Automattic/wp-calypso/assets/5039531/80face21-c779-4c6a-8bc4-fbf1d385e6d9">

Reply:
This is a great idea, ty.

I've implemented it at https://github.com/Automattic/wp-calypso/pull/78018/commits/c9c94345dac6275b4d39bf5f1991fe01018c6a41